src/processes/helper/preAutoWelcomeMsg.js

```diff
@@ -6,13 +6,9 @@ const {LogError} = require('../../databases/models');
 const preAutomateWelcomeMsgProcess = async (returnPrepopulatedDm) => {
   try {
     let currentTime = momentTz().tz('America/Los_Angeles');
-    let getCurrentTimeHour = currentTime.hours();
 
     const randomTime = sample([6, 7, 8, 9]);
```

Comment:
it supposed to be `[6, 7, 8]` because as in task description is between 6:00-9:00

Reply:
cmiiw, karna ini hanya di set jamnya saja jadi memang jam 9 juga harus dimasukan.

Reply:
@falhar @n-permana  jadi ini smpai 8 atau 9 nih ? wkwkwk

Reply:
misal sekarang jam 6:59, kalau set jam doang kan bisa jadi 9:59 yg mana itu lebih dari 9:00

Reply:
Oh iya, tadinya saya kira menitnya selalu 00.. thanks @falhar